Assets/Scripts/Control&Input/ItemSelectSlot.cs

```diff
@@ -110,15 +143,24 @@ private void ChangeItemDisplayed(int nextIndex, bool animate = true)
         selectedIndex = nextIndex;
 
         label.text = SelectedItem != null ? SelectedItem.displayName : "None";
-
-        var position = originalItemHolderPosition + Vector3.up * selectedIndex * spacing;
-        if (animate)
+        for (int i = 0; i < augmentModels.Count; i++)
         {
-            LeanTween.moveLocal(itemHolder.gameObject, position, .2f).setEaseInOutBounce();
+            augmentModels[i].Item1.SetActive(false);
+            if (i == selectedIndex || i == Mod((selectedIndex + 1), itemSlots.Length) || i == Mod((selectedIndex - 1), itemSlots.Length))
+            {
+                augmentModels[i].Item1.SetActive(true);
+            }
+            LeanTween.move(augmentModels[i].Item1, itemSlots[Mod((selectedIndex - i), itemSlots.Length)].transform.position, .2f).setEaseInOutBounce();
         }
-        else
+    }
+
+    public static int Mod(float a, float b)
+    {
+        float c = a % b;
+        if ((c < 0 && b > 0) || (c > 0 && b < 0))
         {
-            itemHolder.localPosition = position;
+            c += b;
```

Comment:
No need to overcomplicate this, simply add `b` to `a` to avoid dropping below 0 in the first place. This method should be removed altogether, or at least turned into an extension method.

Reply:
I can change it to an extension method, an actual modulo operator seems like a useful thing to have instead of only a remainder operator (Thanks c# 😅)